backend/django/project/settings.py

```diff
@@ -75,8 +75,12 @@
 
 DATABASES = {
```

Comment:
In place on using environment variables I've figured out another way. Basically creating a settings_local.py that will be put into .gitignore. When pulled onto a machine you would just need to copy and paste from template into newly created settings_local.py and define the variables for the local environment. I tested it on a practice repo and it works. I think this would be a good set up especially if you have multiple django apps?

Reply:
It sounds like `settings_local.py` would work for development, but the disk storage on Heroku is ephemeral, so we can't just upload arbitrary files after deploy and expect them to stay. Is it possible to do a hybrid of these two methods perhaps? `settings_local.py` for development testing and environmental variables for the production push.